code/a/analyze_poverty.do

```diff
@@ -0,0 +1,180 @@
+/* GOAL: Create SHRUG rural poverty analysis dataset */
+
+/* get rural poverty rate and consumption level */
+use shrid secc_pov_rate_tend_rural using $shrug/data/shrug_secc, clear
+ren secc_pov_rate_tend_rural secc_pov_rate_rural
+
+/* get district and subdistrict ids */
+merge 1:1 shrid using $shrug/keys/shrug_pc11_district_key, keepusing(pc11_state_id pc11_district_id) nogen keep(match master)
+merge 1:m shrid using $shrug/keys/shrug_pc11_subdistrict_key, keepusing(pc11_subdistrict_id) nogen keep(match master)
```

Comment:
Note that `$shrug/keys` and `$shrug/data` paths will need to match exactly in the replication data folder. Seeing that this repo hasn't been tested with the test user yet, this is something that may break.

Reply:
the `shrug` global has been set to `repdata` so hopefully this shouldn't create a problem. Will update this once I run the makefile on the test user account.